### PR TITLE
Do not trim trailing whitespace

### PR DESCRIPTION
**Pull request by stevemao**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# editorconfig.org
```
Comment (silvenon) [inline .editorconfig:1]: This is a useful link, I would prefer if you leave it :grin:

**Comment by silvenon** [comment]
Hey :smile: Could you elaborate this change? What's the problem with trimming white space in Markdown files?

File: .editorconfig (hunk 1 above)
Comment (stevemao) [inline .editorconfig:1]: This is to be consistent with other repos. I see this line was removed in other repos.

**Comment by stevemao** [comment]
- to be consistent with other repos
- https://github.com/google/web-starter-kit/commit/4b28eac680d9ee0d656b746ab6239dc47d31a469 would be a good example

**Comment by stevemao** [comment]
Basically two trailing space can generate a new line

File: .editorconfig (hunk 1 above)
Comment (silvenon) [inline .editorconfig:1]: Maybe it was never there in other projects, not intentionally removed. The link is present in generator-gulp-webapp, though. Could you leave it for now? I'll think it's a helpful inconsistency :smiley:

**Comment by silvenon** [comment]
Got it. I'll merge after the comment in the header is added back.

File: .editorconfig (hunk 1 above)
Comment (stevemao) [inline .editorconfig:1]: Sure. I'm not really sure about this. @sindresorhus should know more details.

**Comment by stevemao** [comment]
Done

**Comment by silvenon** [comment]
Thanks!

**Comment by stevemao** [comment]
:smile:

**Pull request merged**